ligand/templates/endogenous_browser.html

```diff
@@ -225,30 +227,32 @@
                         <!-- Receptor block -->
                         <td style="text-align:right">{{value.0}}</td>
                         <td style="text-align:right">{{value.1}}</td>
-                        <td style="text-align:right"><a target='_blank' href="http://www.uniprot.org/uniprot/{{value.18}}">{{value.2}}</a></td> <!-- UniProt -->
-                        <td style="text-align:right"><a target='_blank' href="/protein/{{value.19}}">{{value.3}}</a></td> <!-- IUPHAR -->
+                        <td style="text-align:right"><a target='_blank' href="http://www.uniprot.org/uniprot/{{value.20}}">{{value.2}}</a></td> <!-- UniProt -->
+                        <td style="text-align:right"><a target='_blank' href="/protein/{{value.21}}">{{value.3}}</a></td> <!-- IUPHAR -->
                         <td style="text-align:right">{{value.4}}</td>
 
                         <!-- ligand block -->
-                        <td style="text-align:left"><a href='/ligand/{{value.17}}/info' target='_blank'>{{value.5|title}}</a></td>
+                        <td style="text-align:left"><a href='/ligand/{{value.19}}/info' target='_blank'>{{value.5|title}}</a></td>
                         <td style="text-align:left">{{value.7|title}}</td>
                         <td style="text-align:left" class="pathway">{{value.8|floatformat:0}}</td>
                         <td style="text-align:left" class="pathway ">{{value.9}}</td>
+                        <td style="text-align:left" class="pathway ">{{value.10}}</td>
```

Comment:
Please add django template if else statements to these two lines (SMILES, INCHI key) to show a - when there is no data.